cmd/snap-repair/cmd_run.go

```diff
@@ -54,13 +54,20 @@ func init() {
 		baseurl = "https://api.snapcraft.io/v2/"
 	}
 
+	// allow overwriting the SAS url too
```

Comment:
this comment is a a bit misleading nowdays. We never really talk directly to SAS, we just historically have a way to touch only assertion requests

// allow redirecting assertion requests under a different base url